src/scripts/main.js

```diff
@@ -1,7 +1,26 @@
 'use strict';
 
+const createNotificationDiv = (posTop, posRight, type) => {
+  const notification = document.createElement('div');
+
+  notification.classList.add('notification', type);
+  notification.style.top = `${posTop}px`;
+  notification.style.right = `${posRight}px`;
+
+  return notification;
+};
+
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notificationDiv = createNotificationDiv(posTop, posRight, type);
+  const markup = `
+    <h2 class='title' >${title}</h2>
+    <p>${description}</p>
```

Comment:
There is no break after the first part of the description.

#### Your implementation:
![image](https://user-images.githubusercontent.com/87986955/233680110-4e1c74d6-71da-4d85-91c0-a6eec13216e6.png)
#### Desired outcome:
![image](https://user-images.githubusercontent.com/87986955/233680250-3e7ae8af-6308-4035-8f91-c217cc7301b8.png)